docs/src/css/custom.css

```
@@ -5,26 +5,520 @@
+  --secondary-blue-900: #c6d6ff;
+  --sidebar-bg-color: #161f36;
+  --primary-neutral-800: #c9c9cc;
+  --ifm-button-background-color: #25c2a0;
+  --ifm-button-background-color-dark: #2e8555;
+  --ifm-navbar-link-color: #9da4ae;
+  --brand-color: white;
+  --primary-neutral-600: #c4c4c4;
+  --next-prev-border-color: #293441;
+}
+
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Mono&display=swap');
```

Biome (1.9.4)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)